modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/thunks/fieldChange.es.js

```diff
@@ -32,6 +32,10 @@ const getEditedPages = ({editingLanguageId, name, pages, value}) => {
 						...field.localizedValue,
 						[editingLanguageId]: value,
 					},
+					localizedValueEdited: {
+						...(field.localizedValueEdited ?? {}),
```

Comment:
I have a question about this line, Is it necessary to check if the `field.localizedValueEdited` is true to render an empty object?

Reply:
@interaminense no, it's not necessary to do that. The propose of `localizedValueEdited` is store on which language a field was edited in order to decide if we use the values in default language or not.